### DISCUSSION TITLE
localhost:5000 not responding and no cameras connecting

### DISCUSSION THREAD

**Discussion by recoveringlawyer** [mrlt8/docker-wyze-bridge#1233] [Q&A]
I returned from a week away and found that my WB is not working. When I try to open localhost:5000 I get the login page but my Wyze credentials don't work - it says there is no response from the requested page. I suspect this has something to do with APIs, but the info on that topic in this forum is totally overwhelming. I have no Linux knowledge or experience - I can handle FORTRAN, Basic or C++ or even COBOL (showing my age, I know) but I can't apply any of that to the Docker world! Can somebody point this technological "left-behind" to step-by-step instructions with sample scripts that will help me get things going again? I'll greatly appreciate any assistance!!

**Comment by mrlt8** [comment]
Wyze changed their authentication method again, so we're using the wyze approved API Key/ID method: https://support.wyze.com/hc/en-us/articles/16129834216731-Creating-an-API-Key

You should then be able to update your `docker-compose.yml` to include that:
```yaml
            - WYZE_EMAIL=
            - WYZE_PASSWORD= 
            - API_ID=
            - API_KEY= 
```

**Reply by recoveringlawyer** [reply]
Progress! But a new wrinkle. Using the following compose.yml:

services:
  wyze-bridge:
    container_name: wyze-bridge
    restart: unless-stopped
    image: mrlt8/wyze-bridge:latest
    ports:
      - 8554:8554
      - 8888:8888 
      - 5000:5000 
    environment:           
      - WYZE_Email=xxxxxx
      - WYZE_PASSWORD=xxxxxxx
      - API_ID=xxxxxx
      - API_KEY=xxxxxxxx

it uses DEFAULT WB_API, and I receive a warning that credentials are not set. I can go to the WebUI and set the credentials, but when I try to open localhost:5000 I am told to sign in (never saw this before) but it will not accept any username/password that I can think of. I tried deleting my API and obtaining a new one, but got the same result. Along with my litany of programming inadequacies I have to confess that I have not been a Windows user for over 20 years (only bought this Windows 10 box to run Blueiris). Did I mess something up in Windows?

**Reply by mrlt8** [reply]
I know it's a little confusing, but some users were running the bridge on open servers, so authentication is enabled by default starting with v2.9. 

It should default to your your wyze email and password, or you can manually change it by setting the environment variables  `- WB_USERNAME=myusername` and `- WB_PASSWORD=mypassword`.

The streams will also require a username and password when authentication is enabled. The user name should be `wb` and the password will be a random string that will be displayed at the bottom of the web-ui, or you can manually set it with the environment variable `- WB_API=mystreampassword`.

If you're running the bridge 100% local, then you should be able to disable the web-ui login and stream authentication by adding the environment variable  `- WB_AUTH=false` to your compose.

**Comment by recoveringlawyer** [comment]
Got it! Thanks!

On May 27, 2024, at 10:03 PM, mrlt8 ***@***.***> wrote:



Wyze changed their authentication method again, so we're using the wyze approved API Key/ID method: https://support.wyze.com/hc/en-us/articles/16129834216731-Creating-an-API-Key

You should then be able to update your docker-compose.yml to include that:

            - WYZE_EMAIL=
            - WYZE_PASSWORD=
            - API_ID=
            - API_KEY=

**Comment by recoveringlawyer** [comment]
That did it! Thank you so much!

On May 30, 2024, at 8:42 PM, mrlt8 ***@***.***> wrote:



I know it's a little confusing, but some users were running the bridge on open servers, so authentication is enabled by default starting with v2.9.

It should default to your your wyze email and password, or you can manually change it by setting the environment variables - WB_USERNAME=myusername and - WB_PASSWORD=mypassword.

The streams will also require a username and password when authentication is enabled. The user name should be wb and the password will be a random string that will be displayed at the bottom of the web-ui, or you can manually set it with the environment variable - WB_API=mystreampassword.

If you're running the bridge 100% local, then you should be able to disable the web-ui login and stream authentication by adding the environment variable - WB_AUTH=false to your compose.
